book/src/users/use/ethereum_data.md

```diff
@@ -0,0 +1,17 @@
+# Ethereum data
+
+If Trin was started the HTTP option for web3 access, you can test it by making direct requests.
+For example, get the latest block number.
+```json
+{"jsonrpc": "2.0", "method": "eth_blockNumber", "params": [], "id":1}
```

Comment:
`Eth` endpoints are not currently implemented but we will have to reimplement them again soon.

Reply:
I have updated the description to imply this intent